src/MapReduceResult.php

```diff
@@ -71,7 +71,7 @@ public function getCounts()
      */
     public function getExecutionTimeMS()
     {
-        return $this->executionTimeMS;
+        return (integer) $this->executionTimeMS;
```

Comment:
Is this redundant in light of `$this->executionTimeMS = (integer) $result->timeMillis;` in the constructor? No objections if you want to keep this as a sanity check.

I realize we're at a disadvantage since we can't yet use type hints for class properties.

Reply:
It's redundant, but makes it more the same with other places. I'm just going to keep it.